### PR TITLE
Applied to Win Server 2019

### PR DESCRIPTION
**Pull request by joinimran**
As this approach is also applicable on Windows Server 2019 so updating this accordingly. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9375

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-firewall/isolated-domain.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ ms.technology: mde
 **Applies to**
 -   Windows 10
 -   Windows Server 2016
+-   Windows Server 2019
```
Comment (illfated) [inline windows/security/threat-protection/windows-firewall/isolated-domain.md:26]: General codestyle improvement suggestion:
```suggestion
**Applies to:**
- Windows 10
- Windows Server 2016
- Windows Server 2019
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Only @illfated's suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**